group_vars/location_perleberger36/general.yml

```diff
@@ -0,0 +1,3 @@
+---
+contact_nickname: 'Perry'
+contact_email: 'sprotejesvalkata@gmail.com'
```

Comment:
Perry's address is <isprotejesvalkata@gmail.com>. Please change that on the deployed site too.